src/app/page.tsx

```diff
@@ -0,0 +1,61 @@
+import Link from "next/link"
+
+const CHTRID = 5;
+
+
+export default function Page(){
+    return (
+        <div>
+            <h1>StarWars character table</h1>
+            <h2>The main goal is to create app to fetch StarWars characters from API. (GraphQl)</h2>
+            <p>Show them in a table, and access their information by clicking on the table</p>
+            <Link href={`/character/${CHTRID}`}>Character {CHTRID}</Link>   
+        </div>
+    )
+}
+
+
+/**
+ * 
+ * {
```

Comment:
We can move this to a `.json` file or we can create a constant with this data for now

```js
const data = {
 ...
}
```